### PR TITLE
fix(argo-cd): redis and redis-exporter image was migrated from docker hub to public ecr for docker hub rate limit

### PR DESCRIPTION
**Pull request by kahirokunn**
Note on DCO:

If the DCO action in the integration test fails, one or more of your commits are not signed off. Please click on the *Details* link next to the DCO action for instructions on how to resolve this.

Checklist:

* [x] I have bumped the chart version according to [versioning](https://github.com/argoproj/argo-helm/blob/main/CONTRIBUTING.md#versioning)
* [x] I have updated the documentation according to [documentation](https://github.com/argoproj/argo-helm/blob/main/CONTRIBUTING.md#documentation)
* [x] I have updated the chart changelog with all the changes that come with this pull request according to [changelog](https://github.com/argoproj/argo-helm/blob/main/CONTRIBUTING.md#changelog).
* [x] Any new values are backwards compatible and/or have sensible default.
* [x] I have signed off all my commits as required by [DCO](https://github.com/argoproj/argoproj/blob/master/community/CONTRIBUTING.md).
* [x] My build is green ([troubleshooting builds](https://argo-cd.readthedocs.io/en/stable/developer-guide/ci/)).

Changes are automatically published when merged to `main`. They are not published on branches.

### CODE REVIEW

**Comment by yu-croco** [comment]
@kahirokunn 
Thank you for your PR!
Could you please update the change log, the line 24 on `charts/argo-cd/Chart.yaml` ? 🙋 

> I have updated the chart [changelog](https://github.com/argoproj/argo-helm/blob/main/CONTRIBUTING.md#changelog) with all the changes that come with this pull request according to changelog.

**Comment by kahirokunn** [comment]
@yu-croco Thx. Done.

File: charts/argo-cd/Chart.yaml
Comment (yu-croco) [inline charts/argo-cd/Chart.yaml:24]: @kahirokunn 
Please delete old one 🙏

Comment (kahirokunn) [reply charts/argo-cd/Chart.yaml]: Thx! I got it!

**Review by yu-croco — APPROVED** [review]
Thank you for your contribution! LGTM.
Let me give other maintainers some time, just in case, before merging. 🙏

**Review by mbevc1 — APPROVED** [review]

**Pull request merged**
